### PR TITLE
 app-text/master-pdf-editor: version bump to 5.1.30 and 5.1.36.

### PR DESCRIPTION
**Pull request by jpataias**
Removed 4.3.82, 5.0.36 and 5.1.00 as sources are no longer available.
Version bump to 5.1.30.

### CODE REVIEW

File: app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils desktop
+
+DESCRIPTION="Master PDF Editor is a complete solution for viewing and editing PDF files"
```
Comment (Zlogene) [inline app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild]: you should not out a package name  to the `DESCRIPTION` I've already fixed this in your package like twice.

File: app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
```
Comment (Zlogene) [inline app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild]: `-> ${P}.tar.gz`

Comment (mgorny) [reply app-text/master-pdf-editor/master-pdf-editor-5.1.30.ebuild]: Why? The distfile name is properly versioned already. Renaming it gives no benefit. Please stop forcing that everywhere for no reason.

**Comment by jpataias** [comment]
Just reordered the commits and  changed the description.

Anything else?

Thanks!

**Pull request closed without merging**